hardware_layer/hardware_arduino/src/thruster.cpp

```diff
@@ -0,0 +1,69 @@
+#include <iostream.h>
+#include "thrustercalibration.h"
+
+#define HIGH 3
+#define MED 2
+#define LOW 1
+
+using namespace std;
+
+thruster::thruster();
+
+void thruster::mode(int P)
+{
+    if(P==0)
+        return 0;
+    else if(P<180)
+        return LOW;
+    else if(pwm>=180 && pwm<220)
+        return MED;
+    else
+        return HIGH;
+}
+
+void thruster::setValues(int HC, int MC, int LC, int HAC, int MAC, int LAC)
+{
+    clock_.High=HC;
```

Comment:
To make expressions look clean, it is recommended that spaces are used often. For instance, `clock_.High = HC;`

Reply:
Done